docs/contract_metadata.md

```diff
@@ -1,28 +1,33 @@
 Build Artifact
 ==============
 
-When a compilation succeeds, Remix creates a JSON file for each compiled contract.
-The JSON file contains the compilation's artifact.  Actually, for this to happen, you need to check the **Generate contract metadata** box in the **General settings** section of the **Settings** module.  The .JSON file with the metadata will appear in the file explorer where the compiled file is located.
+When a compilation succeeds, Remix creates two JSON files for each compiled contract.  One of these files **captures the output from the Solidity compilation**. This file is named with the suffix _metadata.json - which is to say this metatdata file generated from **filename.sol** will be **filename_metadata.json** .
```

Comment:
> is to say this metatdata file generated from **filename.sol** will be **filename_metadata.json** .

this is by contract name.
if there are 2 contracts in a file named `test1` and `test2`, `test1_metadata.json` and `test2_metadata.json` are generated.